protecode/model.py

```diff
@@ -46,12 +46,15 @@ def name(self):
     def vulnerabilities(self) -> 'Iterable[Vulnerability]':
         return (Vulnerability(raw_dict=raw) for raw in self.raw.get('vulns'))
 
-    def highest_major_cve_severity(self) -> int:
+    def highest_major_cve_severity(self, ignore_triage:bool = False) -> int:
```

Comment:
convert to static method; pass-in vulnerabilities. apply "determine highest cve" logic to passed-in sequence / iterable of vulns (remove new flag)